src/content/docs/en/guides/integrations-guide/tailwind.mdx

```diff
@@ -159,6 +159,10 @@ export default defineConfig({
 
 You can now [import your own `base.css` as a local stylesheet](/en/guides/styling/#import-a-local-stylesheet).
 
+#### nesting
+
+Applies the [`tailwindcss/nesting` PostCSS plugin](https://tailwindcss.com/docs/using-with-preprocessors#nesting) so you can use Tailwind's syntax with CSS nesting.
```

Comment:
@bluwy  Would a code sample here look identical to the one above, just showing `nesting` instead of `applyBaseStyles: false`? Is this a boolean? Can it be toggled off? What's the default?

Reply:
Sorry, I wrote this in a whim and it indeed looks vague looking back now 😅 Yes it's a boolean you can enable and it's false by default. Will polish this up tomorrow with code examples too!

Reply:
Haha, perfect! Thanks, no rush! Just wanted to clarify!